fastly/data_source_fastly_tls_domain.go

```diff
@@ -0,0 +1,133 @@
+package fastly
+
+import (
+	"fmt"
+
+	"github.com/fastly/go-fastly/v3/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceFastlyTLSDomain() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceTLSDomainsRead,
```

Comment:
As per the other PR: let's keep the naming of functions consistent so in this case:
```suggestion
		Read: dataSourceFastlyTLSDomainsRead,
```

If you could apply this throughout the PR please.